### PR TITLE
[WWST-7126] Fingerprint for Somfy Situo 1

### PR DESCRIPTION
**Pull request by MWierzbinskaS**
@SmartThingsCommunity/srpol-pe-team 

@greens could you please take a look at the code?

### CODE REVIEW

File: devicetypes/smartthings/ikea-button.src/ikea-button.groovy [hunk 1]
```diff
@@ -366,7 +367,7 @@ private boolean isIkea() {
 }
 
 private boolean isSomfySituo() {
-	device.getDataValue("model") == "Situo 4 Zigbee"
+	device.getDataValue("model") == "Situo 4 Zigbee" || device.getDataValue("model") == "Situo 1 Zigbee"
```
Comment (greens) [inline devicetypes/smartthings/ikea-button.src/ikea-button.groovy]: Do you want to just refactor this to `isSomfy` and just check the manufacturer?

File: devicetypes/smartthings/ikea-button.src/ikea-button.groovy [hunk 2]
```diff
@@ -32,6 +32,7 @@ metadata {
 		fingerprint manufacturer: "IKEA of Sweden", model: "TRADFRI open/close remote", deviceJoinName: "IKEA Remote Control", mnmn: "SmartThings", vid: "SmartThings-smartthings-IKEA_TRADFRI_open/close_remote" // raw description 01 0104 0203 01 07 0000 0001 0003 0009 0020 1000 FC7C 07 0003 0004 0006 0008 0019 0102 1000 //IKEA TRÅDFRI Open/Close Remote
 		fingerprint manufacturer: "KE", model: "TRADFRI open/close remote", deviceJoinName: "IKEA Remote Control", mnmn: "SmartThings", vid: "SmartThings-smartthings-IKEA_TRADFRI_open/close_remote" // raw description 01 0104 0203 01 07 0000 0001 0003 0009 0020 1000 FC7C 07 0003 0004 0006 0008 0019 0102 1000 //IKEA TRÅDFRI Open/Close Remote
 		fingerprint manufacturer: "SOMFY", model: "Situo 4 Zigbee", deviceJoinName: "SOMFY Remote Control", mnmn: "SmartThings", vid: "SmartThings-smartthings-Somfy_open/close_remote" // raw description 01 0104 0203 00 02 0000 0003 04 0003 0005 0006 0102
+		fingerprint manufacturer: "SOMFY", model: "Situo 1 Zigbee", deviceJoinName: "SOMFY Remote Control", mnmn: "SmartThings", vid: "SmartThings-smartthings-Somfy_open/close_remote" // raw description 01 0104 0203 00 02 0000 0003 04 0003 0005 0006 0102
```
Comment (greens) [inline devicetypes/smartthings/ikea-button.src/ikea-button.groovy:35]: what's the difference between 4 and 1? Is there any?

File: devicetypes/smartthings/ikea-button.src/ikea-button.groovy (hunk 1 above)
Comment (MWierzbinskaS) [reply devicetypes/smartthings/ikea-button.src/ikea-button.groovy]: Done

File: devicetypes/smartthings/ikea-button.src/ikea-button.groovy (hunk 2 above)
Comment (MWierzbinskaS) [reply devicetypes/smartthings/ikea-button.src/ikea-button.groovy:35]: Those are very similar devices. I think that here you can find good explanation about differences https://www.screeninnovations.com/accessories/situo-remote-4-ch/

**Review by dkirker — APPROVED** [review]

File: devicetypes/smartthings/ikea-button.src/ikea-button.groovy (hunk 1 above)
Comment (dkirker) [reply devicetypes/smartthings/ikea-button.src/ikea-button.groovy]: I suppose naming the function `isSomfy` or leaving as `isSomfySituo` is fine, as they are both accurate.

**Review by greens — APPROVED** [review]

**Pull request merged**